tests/framework/helpers/ArrayHelperTest.php

```diff
@@ -1269,5 +1269,15 @@ public function testFilter()
         $this->assertEquals(ArrayHelper::filter($array, ['X']), []);
         $this->assertEquals(ArrayHelper::filter($array, ['X.Y']), []);
         $this->assertEquals(ArrayHelper::filter($array, ['A.X']), []);
+        
+        $tmp = [
+            'a' => 0,
+            'b' => '',
+            'c' => false,
+            'd' => null,
+            'e' => true
+        ];
+        
+        $this->assertEquals(ArrayHelper::filter($tmp, array_keys($tmp)),$tmp);
```

Comment:
@hhniao I can also propose to add here space after comma, like that:
```
$this->assertEquals(ArrayHelper::filter($tmp, array_keys($tmp)), $tmp);
```
It's because of PSR-2.